Sprint-1/errors/3.js

```diff
@@ -1,9 +1,17 @@
-const cardNumber = 4533787178994213;
-const last4Digits = cardNumber.slice(-4);
+let cardNumber = 4533787178994213;
```

Comment:
Can this be move to the line preceding the rest of your code.

This will help with readability of your code.